qa/tasks/cephfs/test_volumes.py

```diff
@@ -9122,3 +9122,51 @@ def test_volumes_module_finisher_thread(self):
 
         # verify trash dir is clean
         self._wait_for_trash_empty()
+
+class TestEmptyStringForCreates(CephFSTestCase):
+    CLIENTS_REQUIRED = 1
+    MDSS_REQUIRED = 1
+
+    def _fs_cmd(self, *args):
+        return self.mgr_cluster.mon_manager.raw_cluster_cmd("fs", *args)
```

Comment:
Not really need, calling `self.run_ceph_cmd()` is already available and that will suffice.

Reply:
rewrote command invocation with self.run_ceph_cmd()